### PR TITLE
Replace `AvatarIcon` with `LucideIcon`.

### LINKED ISSUE #3525: Replace `AvatarIcon` with `LucideIcon`.
### Description

Remove `AvatarIcon` in `Avatar` and use `@yamada-ui/lucide` icon.

### Problem Statement/Justification

To unify with `@yamada-ui/lucide` and remove unnecessary icons。

### Proposed Solution or API

_No response_

### Alternatives

_No response_

### Are you willing to participate in implementing this feature and create a pull request include the implementation?

None

### Additional Information

_No response_

### PR DESCRIPTION
**Pull request by illionillion**
Closes #3525  

## Description

Remove `AvatarIcon` in `Avatar` and use `@yamada-ui/lucide` icon.

### CODE REVIEW

**Review by kaehehehe — DISMISSED** [review]

**Review by kaehehehe — COMMENTED** [review]
@illionillion 
I'm sorry for asking this after the approval. I have a simple question. Why did you decide to use the Lucide icon directly in `avatar.tsx` instead of replacing the Icon in `avatar-icon.tsx`? I thought that using React Dev Tools would make it difficult to search for `AvatarIcon` since it wouldn't have a displayName. Is that not a concern?

**Comment by illionillion** [comment]
> @illionillion I'm sorry for asking this after the approval. I have a simple question. Why did you decide to use the Lucide icon directly in `avatar.tsx` instead of replacing the Icon in `avatar-icon.tsx`? I thought that using React Dev Tools would make it difficult to search for `AvatarIcon` since it wouldn't have a displayName. Is that not a concern?

@kaehehehe 
Because Mr. Yamada said he would delete it.
The `v2` modification will apply `displayName` to all the icons in `@yamada-ui/lucide`, so that should be no problem.

**Review by hirotomoyamada — CHANGES_REQUESTED** [review]
Insert `@yamada-ui/lucide` to `package.json`.

**Comment by illionillion** [comment]
@hirotomoyamada 
I fixed `package.json`.
Please review again.

File: packages/components/avatar/src/avatar.tsx [hunk 1]
```diff
@@ -10,7 +10,7 @@ import {
 import { useImage } from "@yamada-ui/image"
 import { createContext, cx, dataAttr, handlerAll } from "@yamada-ui/utils"
 import { cloneElement, useState } from "react"
-import { AvatarIcon } from "./avatar-icon"
+import { UserRoundIcon } from "../../lucide/src"
```
Comment (kaehehehe) [inline packages/components/avatar/src/avatar.tsx:13]: I think you need to fix this line.
```suggestion
import { UserRoundIcon } from "@yamada-ui/lucide"
```

Comment (illionillion) [reply packages/components/avatar/src/avatar.tsx]: FIxed.

**Review by kaehehehe — DISMISSED** [review]
LGTM!

**Review by hirotomoyamada — APPROVED** [review]

**Pull request merged**